pkg/protocol/suite/server.go

```diff
@@ -79,21 +96,34 @@ func (c *Config) Load(core Core, protocol string) (server protocol.Server, err e
 	return c.configMap[protocol].New(core)
 }
 
-func (c *Config) LoadAll(core Core) (serverMap ServerMap, err error) {
+func (c *Config) LoadAll(core Core) (serverMap ServerMap, streamServerMap StreamServerMap, err error) {
 	serverMap = make(ServerMap)
 	var server protocol.Server
-	for protocol := range c.configMap {
-		if server, err = c.configMap[protocol].New(core); err != nil {
-			return nil, err
+	for proto := range c.configMap {
+		if server, err = c.configMap[proto].New(core); err != nil {
```

Comment:
Here "streamServerMap" has not been processed, while "streamServerMap" can be parsed normally

Reply:
Sorry, don't get this.. 
Do you mean return `streamServerMap` here? It will be `nil`. Just like `return nil, nil, err`.

Reply:
Yes~

- If configMap has error, the 'streamServerMap' always be nil.
- If streamConfigMap has error, the 'serverMap' always not be nil.
Although there's nothing wrong with writing it this way, I suggest returning "nil, nil, err" for all

Reply:
done